inc/dashboard-widgets/namespace.php

```diff
@@ -86,9 +84,13 @@ function get_community_events_ajax() : void {
  * @return array|WP_Error List of events or WP_Error on failure.
  */
 function get_community_events() {
-	$response = wp_remote_get( EVENTS_API );
-	if ( is_wp_error( $response ) ) {
-		return $response;
+	$response = wp_cache_get( EVENTS_API );
```

Comment:
We should use the Transient API here instead. Sites without a persistent cache won't see a benefit when this isn't likely to get hit many times in one page load.

Reply:
Actually, even with a persistent cache `WP_Cache_Object` lasts for a page load. It's subsequent page loads that are affected.

Reply:
That's what I mean - the non-persistent cache only lasts for a page load, so using wp_cache_* wouldn't give any benefit for those sites, so the Transient API is better here